v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,44 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT
+    sub.question_id,
+    avg(sub.mean_score_per_user_per_question) as mean_score_per_question,
+    sub.number,
+    corr(sub.mean_score_per_user_per_question, sub2.average_assessment_score_for_user) as discrimination
+FROM (
+    SELECT
+        assessment_instances.user_id,
+        assessment_questions.question_id,
+        assessment_questions.number,
+        avg(instance_questions.points/assessment_questions.max_points) as mean_score_per_user_per_question
+    FROM
+        instance_questions
+        JOIN assessment_instances ON (instance_questions.assessment_instance_id=assessment_instances.id)
+        JOIN assessment_questions ON (instance_questions.assessment_question_id=assessment_questions.id)
+    WHERE
+        assessment_questions.assessment_id = $assessment_id AND
+        assessment_questions.deleted_at IS NULL
+    GROUP BY
+        user_id,
+        question_id,
+        assessment_questions.number
+) sub
+    JOIN (
+        SELECT
+            assessment_instances.user_id,
+            avg(assessment_instances.score_perc/100.0) as average_assessment_score_for_user
```

Comment:
I know you are doing the `/100` here for consistency with average score, but I would suggest making everything `score_perc` (including the average score) because that's what we are using everywhere else. I want to keep the codebase standardized on `score_perc` and `points` as the two standard measurements.